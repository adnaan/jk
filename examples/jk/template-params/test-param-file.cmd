jk run ./template.js -f test-param-file.yml -p port=8888
