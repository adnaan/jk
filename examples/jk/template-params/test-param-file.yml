selector:
  application: jkats
  version: 1.3.1
name: jkats-service
port: 9000
