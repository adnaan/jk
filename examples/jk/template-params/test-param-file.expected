apiVersion: v1
kind: Service
metadata:
  name: jkats-service
  namespace: default
spec:
  ports:
  - port: 8888
  selector:
    application: jkats
    version: 1.3.1
